src/components/taskDetails/Details.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import classNames from './task-details.module.scss';
+
+function setDetailsColor(value: string = '') {
+  switch (value) {
+    case 'HIGH':
+      return 'red';
+    case 'MEDIUM':
+      return 'orange';
+    case 'LOW':
+      return 'green';
+    default:
+      return 'black';
+  }
+}
```

Comment:
hey the color is set to priority only, shouldn't this be `setPriorityColor`

Reply:
discussed